user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -4,25 +4,80 @@ uid: Swarming
+
+Below you can find a complete overview of the differences between a system using DataMiner 10.5.1 or higher with or without Swarming.
+
+|                                                  | 10.5.1+&nbsp; systems without Swarming | 10.5.1+ systems with Swarming  |
+|--------------------------------------------------|----------------------------------------|--------------------------------|
+| Element configuration                            | On disk (element.xml)                  | In database                    |
+| Alarm IDs                                        | Per DataMiner Agent                    | Per element                    |
+| Database per DMA                                 | Supported                              | Not supported                  |
+| Scripts & QActions using legacy alarm references | Supported                              | Not supported                  |
+| Legacy Reports & Dashboards                      | Supported                              | Not supported                  |
+| Failover                                         | Supported                              | Not supported                  |
+| Legacy Correlation                               | Not supported                          | Not supported                  |
+| Offload Database                                 | Supported                              | Not supported*                 |
+| SLA elements                                     | Supported                              | Supported but not swarmable*   |
+| Enhanced services                                | Supported                              | Supported** but not swarmable* |
+| Spectrum elements                                | Supported                              | Supported but not swarmable*   |
+| Redundancy group elements                        | Supported                              | Supported but not swarmable*   |
+| Elements exporting DVEs of functions             | Supported                              | Supported but not swarmable*   |
+| EPM elements                                     | Supported                              | Supported but not swarmable*   |
+| Elements polling localhost                       | Supported                              | Supported but not swarmable    |
+| Elements with element connections                | Supported                              | Supported but not swarmable*   |
+| Smart-serial elements in server mode             | Supported                              | Supported but not swarmable    |
+| Elements receiving SNMP traps                    | Supported                              | Supported but not swarmable    |
+
+(*) In initial release.
```

Comment:
perhaps we should reword this to 
(*) To be added later

Because now you might expect it to be added in 10.5.2 or something?

Reply:
Updated